tasks/eth/006-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,335 @@
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **Meaning:** Sets `l2BlockTime` at slot `0x05` (5). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L45-L49).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000006` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x0000000000000000000000009ba6e03d8b90de867373db8cf1a58d2f7f006b3a` <br/>
+  **Meaning:**  Sets `challenger` at slot `0x06` (6). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L52-L56).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000007` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000473300df21d047806a082244b417f96b32f13a33` <br/>
+  **Meaning:** Sets `proposer` at slot `0x07` (7). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L59-L63).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000008` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000093a80` <br/>
+  **Meaning:**  Sets `finalizationPeriodSeconds` at slot `0x08` (8). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L66-L70).
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x000000000000000000000000db5d932af15d00f879cabebf008cadaaaa691e06` <br/>
+  **After:** `0x000000000000000000000000f243bed163251380e78068d317ae10f26042b292` <br/>
+  **Meaning:** Implementation address is set to the new `L2OutputOracle`.
+
+All other state changes (1) are a nonce change of `0x07dc0893cafbf810e3e72505041f2865726fd073`.
```

Comment:
What does `(1)` mean here?

Reply:
it means that there's only one change, but just improved the wording to make it less confusing